File: core/src/main/java/kandroid/observer/kcsapi/api_req_mission.java
package kandroid.observer.kcsapi;

import kandroid.observer.ApiBase;
import kandroid.observer.RawData;

public class api_req_mission {

    public static class start extends ApiBase {
        @Override
        protected void onDataReceived(RawData rawData) {

        }

        @Override
        public String getApiName() {
            return "api_req_mission/start";
        }
    }

    public static class result extends ApiBase {

        @Override
        protected void onDataReceived(RawData rawData) {

        }

        @Override
        public String getApiName() {
            return null;
        }
    }
}

File: app/src/main/java/kandroid/observer/kcsapi/api_req_mission.kt
package kandroid.observer.kcsapi

import kandroid.observer.ApiBase
import kandroid.observer.RawData

object api_req_mission {

    object start : ApiBase() {
        override val name: String get() = "api_req_mission/start"

        override fun onDataReceived(rawData: RawData) {
        }
    }

    object result : ApiBase() {
        override val name: String get() = "api_req_mission/result"

        override fun onDataReceived(rawData: RawData) {
        }
    }
}
